examples/kotlin-matter-controller/java/src/com/matter/controller/Main.kt

```diff
@@ -0,0 +1,63 @@
+/*
+ *   Copyright (c) 2023 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+package com.matter.controller
+
+import com.matter.controller.commands.common.*
+import com.matter.controller.commands.pairing.*
```

Comment:
nit: consider following the android style of "wildcard imports are not allowed":

https://developer.android.com/kotlin/style-guide#import_statements

Reply:
acked